lib/screens/event_details.dart

```diff
@@ -133,42 +133,51 @@ class _EventDetailsScreenState extends ConsumerState<EventDetailsScreen> {
                 .toList()));
   }
 
-  Future<void> _saveRsvp(InstanceMemberStatus? status) async {
-    final eventRsvpsController =
-        ref.read(rsvpsPerEventProvider(widget.instanceId).notifier);
+  Future<void> _saveRsvp(
```

Comment:
@IguJl15 looks like these changes are just a bunch of editor auto-reformatting? Let's lock down `.vscode/settings.json` and anything else so that our editors aren't fighting to reformat things

I can't see if you actually changed anything in here